unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -269,7 +269,7 @@
         <!-- Divider -->
         <!-- <p class="my-4 h-[1px] w-full bg-slate-200" /> -->
 
-        {#if $state?.dev_mode_enabled}
+        {#if $state?.dev_mode != 'Off'}
```

Comment:
Why loose equality here compared to other related changes where it's strict equality?